layouts/custompage/release.html

```diff
@@ -31,7 +31,7 @@ <h2>{{ i18n "the-latest-release" }}</h2>
     {{ if ne .Site.Language.Lang "en" }}
     {{ $langVar = .Site.Language.Lang }}
     {{ end }}
-    {{ range first 1 (sort (where (where .Site.Pages "Section" "release") ".Params.linked" true) ".Date" "desc") }}
+    {{ range first 1 (sort (where (where (where .Site.Pages "Section" "release") ".Params.linked" true) ".File.BaseFileName" .Site.Params.releaseVersion  ) ".Date" "desc") }}
```

Comment:
```suggestion
    {{ range first 1 (sort (where (where (where .Site.Pages "Section" "release") ".Params.linked" true) ".File.BaseFileName" .Site.Params.releaseVersion  ) ".Date" "desc") }}
```

Is the sorting could be skipped? Since only one item is retrieved with latestRelease filtered.

Reply:
done.